moon-store-mac-os/moon-store-mac-os/Services/MSEndpoint.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  MSPath.swift
+//  moon-store-mac-os
+//
+//  Created by Jose Luna on 12/3/24.
+//
+
+import Foundation
+
+enum MSEndpoint {
+    case users
+    case login
+    case updatePassword
+    case products
+    case productCount
+    case register
+    case enableUser
+    case disableUser
+    case roles
+    case usersChart
+    case invoices
+    case backupDatabase
+    case restoreDatabase
+    case listDatabase
+    
+    var path: String {
+        "\(prefixV1)\(endpoint)/"
+    }
+    
+    var prefixV1: String { "/api/v1/" }
+    
+    var endpoint: String {
+        switch self {
```

Comment:
A more clean way is to add the `/` in each case to avoid assuming that the base URL has the final slash